javascript-sdk/examples/save-and-load-authenticated-session/save-and-load-authenticated-session.js

```diff
@@ -0,0 +1,77 @@
+const { wrap, configure } = require('agentql');
+const { chromium } = require('playwright');
+
+(async () => {
+  // Set the AgentQL API key via the `configure` method.
+  require('dotenv').config({ path: '.env.local' })
+  configure({ apiKey: process.env.API_KEY });
```

Comment:
@rachelnabors and I are thinking about removing `configure` line and using `dotenv` line from examples. JS SDK currently supports directly read in AGENTQL_API_KEY as a backup option when retrieving API key. We will ask users to set API key as an environment variable in installation guide.